### DISCUSSION TITLE
Pixi.js in isomorphic React app

### DISCUSSION THREAD

**Discussion by sylvestrevgen** [pixijs/pixijs#7288] [General]
I use React Starter Kit: https://github.com/kriasoft/react-starter-kit
I install Pixi.js and react-pixi-fiber packages and it is not working. After researching, there are problems cause isomorphic server side by Node.js.

Is it possible to use Pixi.js in React isomorphic app? Thanks.

**Comment by bigtimebuddy** [comment]
PixiJS doesn't work server-side yet (pure node.js environments), but we have a plan to fix this. There are attempts out there to get SSR to work using shims for all the DOM bits in Pixi, but these solutions are probably not production quality.
